package/yast2-schema-default.spec

```diff
@@ -18,7 +18,7 @@
 
 Name:           yast2-schema-default
 # Keep versions in sync with yast2-schema-micro
-Version:        4.4.9
+Version:        4.4.10
```

Comment:
just please keep in sycn with version in yast2-schema-micro otherwise packaging won't work as it unpack one tarball with version included.